client/src/components/Proverbs/Addproverbs.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import {
+  Modal, Button, Input, message,
+} from 'antd';
+import './proverb.css';
+import http from '../../services/httpService';
+
+function Addproverbs() {
+  const [visible, setVisible] = useState(false);
+  const [value, setValue] = useState({ proverb: '' });
+  const handleChange = (e) => setValue({ proverb: e.target.value });
+  console.log(value);
+
+  const sendProverb = async () => {
+    try {
+      if (value.proverb !== '') {
+        const proverb = await http.post('api/v1/proverb', value);
+        message.success(proverb.response.data.message);
+      } else {
+        setVisible(true);
+        throw new Error('يجب عليك إضافة مثل');
+      }
+    } catch (error) {
+      message.error(error.response.data.message);
+      console.log(error);
+    }
+  };
+  const handleOpen = () => {
+    setVisible(false);
+    sendProverb();
+  };
+  return (
+    <>
+      <Button type="secondary" onClick={() => setVisible(true)}>
+        إضافة مثل
+      </Button>
+      <Modal
+        title="أضف المثل"
+        centered
+        visible={visible}
+        onOk={() => setVisible(false)}
+        onCancel={() => setVisible(false)}
+        width={1000}
+        footer={null}
+      >
+        <div className="proverbcontainer">
+          <div><Input type="text" onChange={handleChange} className="input" placeholder="أضف هنا المثال" required /></div>
```

Comment:
why you nested an input component with div? you can style an input it took class name